templates/website/src/collections/Pages/index.ts

```diff
@@ -125,7 +125,7 @@ export const Pages: CollectionConfig<'pages'> = {
   hooks: {
     afterChange: [revalidatePage],
     beforeChange: [populatePublishedAt],
-    beforeDelete: [revalidateDelete],
+    afterDelete: [revalidateDelete],
```

Comment:
@paulpopus is this correct? `beforeDelete` doesn't have access to `doc` so ts complains. I don't know if it was working properly before this.

Reply:
Yeah I think this fine if you've tested it works